source/features/unreleased-commits.tsx

```diff
@@ -121,7 +121,7 @@ async function init(signal: AbortSignal): Promise<false | void> {
 
 	await api.expectToken();
 
-	observe(branchSelector, add, {signal});
+	observe(`table[aria-labelledby="folders-and-files"] ${branchSelector}`, add, {signal});
```

Comment:
```suggestion
	observe(`${branchSelector}:not(${branchSelectorParent} ${branchSelector})`, add, {signal});
```
- We can resolve this like this way too.

Reply:
Either way, it should be part of the selector we have saved

Reply:
```js
export const branchSelector = '[data-hotkey="w"]';
export const branchSelectorParent = 'details#branch-select-menu';
```

Since these 2 apis are available currently, It looks hard to me to adding new general selector.
- `${branchSelector}:not(${branchSelectorParent} ${branchSelector})` -> `branchSelectorWithoutBranchSelectorParent` 😶 
  - It looks too long.. and something indescribable.
- `table[aria-labelledby="folders-and-files"] ${branchSelector}` -> `branchSelectorOnNew` 👎 
  - Avoid `new` naming

Or merge `add()` and `addLegacy()`, check is parent `branchSelectorParent` and append with that condition?

I don't know which is more refined way.

Reply:
Let's merge it

Reply:
Never same mistake; Both are checked.

- Disable `New repository overview`
  <img width="208" alt="image" src="https://github.com/refined-github/refined-github/assets/50487467/a14622b9-2ca4-41ac-887b-0860efb05dd6">

- Enable `New repository overview`
  <img width="229" alt="image" src="https://github.com/refined-github/refined-github/assets/50487467/d3ee708f-0e93-4137-bf71-c5e7f4cc5bf8">